src/app/components/sidebar/sidebar.jsx

```diff
@@ -147,17 +161,19 @@ class Sidebar extends React.Component {
           text={TOGGLE_HOTKEY.TEXT}
           onClick={this.toggleSidebarStage}
         />
-        <Resizable
-          ref={this.resizable}
-          mix={b('sidebar__resizable')}
-          direction={direction}
-          initialSize={{ ...initialSize }}
-          maxSize={{ width: maxWidth }}
-          onResize={this.onResize}
-          onResizeStop={this.onResizeStop}
-        >
-          {children}
-        </Resizable>
+        { resizableEnabled ? (
+          <Resizable
+            ref={this.resizable}
+            mix={b('sidebar__resizable')}
```

Comment:
Do we need `b` here? 🤔

Reply:
In some other internal projects we have an agreement to use b whether mods/mix are applied or not. This brings kind of standardization and we are (I am) used to write like this

Reply:
Huh. Okay.